runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldTableRow.java

```diff
@@ -64,12 +62,17 @@
 	private static final Color FREE_WORLD = new Color(200, 200, 200);
 	private static final Color LEAGUE_WORLD = new Color(157, 237, 1);
 
+	private static final Map<WorldRegion, ImageIcon> flags;

```

Comment:
This should follow constant naming conventions.
```suggestion
	private static final Map<WorldRegion, ImageIcon> FLAGS;

```

Reply:
Fixed